addons/ccm-openstack/ccm-openstack.yaml

```diff
@@ -1,16 +1,18 @@
 ---
+# Source: openstack-cloud-controller-manager/templates/serviceaccount.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: cloud-controller-manager
+  name: openstack-cloud-controller-manager
   namespace: kube-system
+  annotations:
```

Comment:
Is there a way that we somehow drop this empty annotations block?

Reply:
yeah, just removing it manually after each `helm template`

Reply:
Okay, that's far from ideal. Let's leave it as it is.